changelogs/fragments/82-changed_true_action_states_check_mode_yes.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - changed flag set for diff when action_states and checkmode yes (https://github.com/ansible-collections/ansible.netcommon/pull/82)
```

I'm not sure how sensible that will be in the middle of a changelog. I'm not sure if this is any better (or if it properly captures the PR), but hopefully it's clearer?
```suggestion
  - Resource Modules: changed flag is set to true in check_mode for all ACTION_STATES (https://github.com/ansible-collections/ansible.netcommon/pull/82)
```

---

looks good, I'll update the PR.